app/models/article.rb

```diff
@@ -374,6 +374,18 @@ def tag_keywords_for_search
     tags.pluck(:keywords_for_search).join
   end
 
+  def edited?
+    edited_at.present?
+  end
+
+  def readable_edit_date
+    if edited_at && edited_at.year == Time.current.year
```

Comment:
I think you can just quit this method so you don't have to check `edited_at`, nor use the `&.` operator.

```ruby
def readable_edit_date
  return unless edited_at
  ...
end
```

I would also add a test :-)